pkgs/applications/editors/vscode/vscode.nix

```diff
@@ -5,6 +5,12 @@
 , nixosTests
 , srcOnly
 , isInsiders ? false
+# sourceExecutableName is the name of the binary in the source archive over
+# which we have no control and it is needed to run the insider version as
+# documented in https://nixos.wiki/wiki/Visual_Studio_Code#Insiders_Build
+# On MacOS the insider binary is still called code instead of code-insiders as
+# of 04-07-2023.
```

Comment:
Nit: Please use the international/ISO standard `YYYY-MM-DD` format here to be unambiguous.

Reply:
Done😃